docker-compose.yml

```diff
@@ -1,27 +1,34 @@
-version: "2"
+version: "3"
 
 services:
   data-loader:
     image: quay.io/keboola/data-loader
     tty: true
     stdin_open: true
     volumes:
-      - ./data:/data/
+      - data:/data/
     environment:
       - KBC_CONFIG_ID
       - KBC_ROW_ID
       - KBC_CONFIG_VERSION
       - KBC_TOKEN
+      - KBC_STORAGEAPI_URL
   sandbox:
-    build: .
+    build:
+      context: .
+      args:
+        NB_USER: ${NB_USER}

```

Comment:
why? shouldn't that be a static value?

Reply:
yes, it was something I was trying, but it doesn't make sense, thanks.